google-books/src/components/BooksLoader/BooksLoader.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import { getBooksBySearch } from "../../services/book-services";
+import BookList from "../BookList/BookList";
+import styles from "./BooksLoader.module.scss";
+import { ColorRing } from "react-loader-spinner";
+import Error from "../Error/Error";
+
+
+
+const BooksLoader = ({ searchTerm }) => {
+	const [books, setBooks] = useState([]);
+	const [error, setError] = useState(null);
+	const [loading, setLoading] = useState(false);
+
+
+	useEffect(() => {
+		setError(null);
+		setBooks([]);
+		setLoading(true);
```

Comment:
`if(searchTerm) ` - make the api call when the search term changes from "" to something, for now you are making one when the page first loads with an empty string as a query